roles/nova-data/templates/etc/serverspec/kvm_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe file('/var/lib/libvirt/images') do
+  it { should be_mode 711 }
+end
+
+describe file('/dev/kvm') do
+  it { should exist }
+  it { should be_mode 660 }
+end 
+
+describe file('/etc/libvirt/libvirtd.conf') do
+  file_contents = ['unix_sock_group = "libvirtd"',
+                   'unix_sock_ro_perms = "0777"',
+                   'unix_sock_rw_perms = "0770"']
```

Comment:
I would say these need to check if the line _starts with_ these items, otherwise it'll catch commented out lines.